@xen-orchestra/lite/src/stores/xen-api/pool.store.ts

```diff
@@ -9,9 +9,12 @@ export const usePoolStore = defineStore('xen-api-pool', () => {
 
   const pool = computed<XenApiPool | undefined>(() => baseContext.records.value[0])
 
+  const isPoolMaster = (hostRef: XenApiPif['host']) => pool.value?.master === hostRef
```

Comment:
Two things here:
- It would be preferable use `XenApiHost['$ref']` than `XenApiPif['host']`?
- `isMasterHost` would probably clarify the intention, since we already are in a "pool" store. What do you think?

```suggestion
  const isMasterHost = (hostRef: XenApiHost['$ref']) => pool.value?.master === hostRef
```

Reply:
Yes, I was wondering about the name